### ISSUE TITLE
Bug in .whois and .match dcc commands after add hosts to user

### ISSUE THREAD

**Issue by VertigoRN** [eggheads/eggdrop#493]
![neufql1hm54](https://user-images.githubusercontent.com/34381201/33780144-c47abdfa-dc60-11e7-93e9-4a8996d64ba4.png)
In .match dcc command this bug also present.
_____
<Vertigo> .status
<Dafna> #Vertigo# status
<Dafna> I am Juliana, running eggdrop v1.8.2: 63 users (mem: 812k).
*****************
<Dafna> Tcl library: /home/Sergey/tcl87/lib/tcl8.7
<Dafna> Tcl version: 8.7a0 (header version 8.7a0)
<Dafna> Tcl is threaded.
<Dafna> TLS support is enabled.
<Dafna> TLS library: OpenSSL 1.0.2k  26 Jan 2017
<Dafna> IPv6 support is enabled.

**Comment by vanosg** [comment]
Hi @VertigoRN ,

 I believe this was fixed as part of #467 . You can download the current develop branch and verify for yourself, but this is what I get:

```
[20:53:01] #-HQ# +user test 
Added test (no host) with no password and no flags.
.tcl setuser test hosts *!*@1.com
Tcl: 
.tcl setuser test hosts *!*@2.com
Tcl: 
.tcl setuser test hosts *!*@3.com
Tcl: 
.tcl getuser test hosts
Tcl: *!*@1.com *!*@2.com *!*@3.com
.whois test
[20:54:12] #-HQ# whois test
HANDLE                           PASS NOTES FLAGS           LAST
test                             no       0 -               never (nowhere)
  HOSTS: *!*@1.com, *!*@2.com, *!*@3.com
```

Thanks so much for taking the time to report this!

**Comment by VertigoRN** [comment]
Yeah. In development version this bug is gone.
```
<Vertigo> .tcl set version
<Dafna> Tcl: 1.8.2+addbot 1080208
<Vertigo> .tcl adduser test_user
<Dafna> Tcl: 1
<Vertigo> .tcl setuser test_user HOSTS aa1!*@*
<Dafna> Tcl: 
<Vertigo> .tcl setuser test_user HOSTS aa2!*@*
<Dafna> Tcl: 
<Vertigo> .tcl setuser test_user HOSTS aa3!*@*
<Dafna> Tcl: 
<Vertigo> .whois test_user
<Dafna> |09.12.2017 / 10:14:02| #Vertigo# whois test_user
<Dafna> HANDLE                           PASS NOTES FLAGS           LAST
<Dafna> test_user                        no       0 -               never (nowhere)
<Dafna>   HOSTS: aa1!*@*, aa2!*@*, aa3!*@*
<Vertigo> .match test_user
<Dafna> |09.12.2017 / 10:14:09| #Vertigo# match test_user
<Dafna> *** Matching 'test_user':
<Dafna> HANDLE                           PASS NOTES FLAGS           LAST
<Dafna> test_user                        no       0 -               never (nowhere)
<Dafna>   HOSTS: aa1!*@*, aa2!*@*, aa3!*@*
<Dafna> --- Found 1 match.
```

Waiting for 1.8.3 :-)
